pkg/sdn/api/validation/validation.go

```diff
@@ -117,3 +117,35 @@ func ValidateNetNamespace(netnamespace *sdnapi.NetNamespace) field.ErrorList {
 func ValidateNetNamespaceUpdate(obj *sdnapi.NetNamespace, old *sdnapi.NetNamespace) field.ErrorList {
 	return validation.ValidateObjectMetaUpdate(&obj.ObjectMeta, &old.ObjectMeta, field.NewPath("metadata"))
 }
+
+// ValidateEgressFirewall tests if required fields in the EgressFirewall are set.
+func ValidateEgressFirewall(egressFirewall *sdnapi.EgressFirewall) field.ErrorList {
+	allErrs := validation.ValidateObjectMeta(&egressFirewall.ObjectMeta, false, oapi.MinimalNameRequirements, field.NewPath("metadata"))
+
+	if egressFirewall.Name != "default" {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("name"), egressFirewall.Name, "EgressFirewall name must be 'default'"))
```

Comment:
Please don't assume that there is only one egress firewall rule in the API.  That breaks other API conventions and assumptions.  If you only honor default from the SDN client, that's fine, but don't make APIs that are "fake" singletons.